tests/test_mtaosCscWithSimulators.py

```diff
@@ -137,16 +138,71 @@ async def test_addAberration_issueCorrection(self):
             # add random values for aberrations
             wf = np.random.rand(19) * 0.1
 
+            dof_before_add_aberration = await remote.evt_degreeOfFreedom.aget(
+                timeout=SHORT_TIMEOUT
+            )
+
+            remote.evt_degreeOfFreedom.flush()
+
             await remote.cmd_addAberration.set_start(wf=wf, timeout=10.0)
 
+            dof_after_add_aberration = await remote.evt_degreeOfFreedom.next(
+                flush=False, timeout=SHORT_TIMEOUT
+            )
```

Comment:
I think the variable names of `dof_add_aberration_before` and `dof_add_aberration_after` might be more clear?